withDom.js

```diff
@@ -1,3 +1,5 @@
+require('raf/polyfill');
```

Comment:
lol yes, sorry. i meant mutating the global outside of DOM-related things, and out of the scope of a withDOM-wrapped test.